conf/app.routes

```diff
@@ -10,6 +10,7 @@ GET        /api/report/:id                            @uk.gov.hmrc.leakdetection
 GET        /api/report/:id/leaks                      @uk.gov.hmrc.leakdetection.controllers.ApiController.reportLeaks(id: ReportId)
 GET        /api/leaks                                 @uk.gov.hmrc.leakdetection.controllers.ApiController.leaks
 GET        /api/leaks/summary                         @uk.gov.hmrc.leakdetection.controllers.ApiController.leaksSummary
+GET        /api/:repository/:branch/report            @uk.gov.hmrc.leakdetection.controllers.ApiController.latestReport(repository: Repository, branch: Branch)
```

Comment:
url might be better off being prefixed as /api/repository/:repo/:branch/report so as not to collide with other urls